src/test/java/com/softserve/kickscooter/kickscootervehicle/management/controller/ManagementEndToEndTest.java

```diff
@@ -0,0 +1,78 @@
+package com.softserve.kickscooter.kickscootervehicle.management.controller;
+
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterCreateDto;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterInfoDto;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
```

Comment:
It's an annotation from JUnit4

Reply:
when you are using `@SpringBootTest`, you don't need neither `@RunWith` nor `@ExtendWith`
take a look at the sources of the `@SpringBootTest` - `@ExtendWith` is already there.